docs/source/entities/maps/explore.md

```diff
@@ -13,6 +13,11 @@ If you can edit the map, a button at the bottom right will show up called **Edit
 
 ![Enter edit mode button](img/map-edit-mode.png)
 
+## Measurement tool
+You can enable the measurement tool in the map settings by filling Distance Unit label and Distance measurement.
+
+image.png
```

Comment:
Please embed the images properly for it to be displayed in the rendered html :)
`![Measurement tool icon](img/map-measurement-tool.jpg)`

Reply:
done ! thank you